internal/server/list_clients.go

```diff
@@ -180,6 +183,32 @@ func renderTemplateForClientTab(client DeputyHubClientInformation, tmpl Template
 			UrlBuilder:      vars.UrlBuilder,
 		}
 
+		if r.Method == http.MethodPost {
+			var err error
+			deputyId, _ := strconv.Atoi(r.PathValue("id"))
+			dueDate := r.FormValue("dueDate")
+			if dueDate == "" {
+				selectDueDateError := sirius.ValidationErrors{
+					"due-date": {"": "Enter a due date"},
+				}
+
+				vars.Errors = util.RenameErrors(selectDueDateError)
+			} else {
+				vars.AppVars = app
+
+				vars.AppVars.SuccessMessage, err = client.BulkAssignAssuranceVisitTasksToClients(ctx, sirius.BulkAssignAssuranceVisitTasksToClientsParams{
```

Comment:
You don't need to reference AppVars here. AppVars is embedded on ListPage, which is embedded in ListClientVars. Embedded fields are fields on the struct in the same way as any other, so you don't need to reference them by way of the struct they are initially a part of. https://gobyexample.com/struct-embedding